script.js

```diff
@@ -1,12 +1,208 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+//Variables
+// Div for showing episodes or movies
+const showEpisodseRow = document.querySelector(".row");
+// Search input
+const searchUser = document.querySelector("#searchUser");
```

Comment:
Why is this called "searchUser"? I would expect that name to imply this field/variable is storing either information about a user who's doing a search, or information about a user being searched for... Can you think of a name that makes more clear what this is for?

_Also_, you have a comment right above which says something very similar - a good rule of thumb is: It's better to explain what something is in the name of the variable, than in a comment. Can you think of a name clear enough that you would be happy to delete the comment and just use the name to know what it is?

Reply:
I change it's name to searchEpisodes and deleted comment, as it is a clear name.